Dockerfile

```diff
@@ -6,20 +6,16 @@
 
 FROM docker.mirror.hashicorp.services/golang:1.16.5-alpine3.13 AS builder
 
-RUN apk add --no-cache git gcc libc-dev openssh make
+RUN apk add --no-cache git gcc libc-dev make
 
 RUN mkdir -p /tmp/wp-prime
 COPY go.sum /tmp/wp-prime
 COPY go.mod /tmp/wp-prime
 
 WORKDIR /tmp/wp-prime
 
-RUN mkdir -p -m 0600 ~/.ssh \
-    && ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
 RUN git config --global url.ssh://git@github.com/.insteadOf https://github.com/
```

Comment:
I think this can go - we can clone over https now.

Reply:
Oops, yeah missed that one! 👍🏻